src/reducers/real/index.js

```diff
@@ -0,0 +1,90 @@
+import {
+  ITEM_ADD,
+  ITEM_DELETE,
+  ITEM_RENAME,
+  CONTAINER_ADD,
+  CONTAINER_DELETE,
+  CONTAINER_RENAME,
+  SET_REAL_DATA
+} from 'actions/actiontypes'
+
+const initialState = {
+  _id: '',
+  name: '',
+  items: [],
+  modified_timestamp: '',
+  containers: [],
+  is_deleted: false,
+  timestamp: '',
+  snapshots: [],
+  unsnapshot_items: []
```

Comment:
I guess i'm just catching this now, but can we not have this? can we instead create a "container" in snapshots called "_unassigned" that is will contain all the unsnapshot items.